internal/testrunner/runners/system/runner.go

```diff
@@ -1530,10 +1531,11 @@ func createPackageDatastream(
 	ds packages.DataStreamManifest,
 	config testConfig,
 ) kibana.PackageDataStream {
+	suffix := createTestRunID()
```

Comment:
Would it be better to just create this suffix (runID) in each create (input or integration) datastream function? Currently, there are no tests of these functions, but having this as a parameter, I think it will be easier to add tests eventually.

Reply:
Or maybe we can pass it from the runner, in case we need to use the runID in more places.

Reply:
:+1: 
In runner it can be set as this suffix `svcInfo.Test.RunID`. That struct is always created.